Fixes #24016 Link points to itself (#24027)
mdn/content: files/en-us/web/css/reference/index.md

## Index

> > **Note:** The property names in this index do **not** include the JavaScript names where they differ from the CSS standard names.

Reviewer: ```suggestion
> **Note:** The property names in this index do **not** include the JavaScript names which do differ from the CSS standard names.
```

> > **Note:** The property names in this index do **not** include the [JavaScript names](https://www.dummies.com/article/technology/programming-web-design/javascript/converting-css-property-names-to-javascript-142516/) where they differ from the CSS standard names.

Reviewer: Thanks. We definitely don't want to include a useless self-referencing link. But, unfortunately, the Dummies site reads it is complete, whereas it is far from complete, so can't be used.

```suggestion
> **Note:** The property names in this index do **not** include the JavaScript names where they differ from the CSS standard names. 
```

A possible useful links include https://developer.mozilla.org/en-US/docs/Web/API/Window/getComputedStyle#notes.

To see a list of names of JS CSS property names, the following code seems to work:

```js
let availableProps = getComputedStyle(document.body);
let list = [];
for(cssProp in availableProps) { 
  if(!/\-/.test(cssProp) && /^[a-z]/.test(cssProp)) {
    list.push(cssProp);
  }
}
console.log(list);
```

See https://codepen.io/estelle/pen/qByJXYg for a list of JS properties available in your current browser.

Author: Okay @estelle

Author: So, shall leave remove the link or add the link you suggested 
https://developer.mozilla.org/en-US/docs/Web/API/Window/getComputedStyle#notes

Author: Are we good to go to merge this @estelle
